src/modules/pmc/src/lib/PmcBase.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "PmcBase.h"
+
+constexpr const char ret[] = R""""({
+    "Name": "PackageManagerConfiguration Module",
```

Comment:
>PackageManagerConfiguration Module [](http://example.com/codeflow?start=13&length=34)

"PackageManagerConfiguration (PMC) Module"?

So we tell users what our short alias is.